sdk/eventhub/azure-eventhub/stress/templates/testjob.yml

```
@@ -0,0 +1,16 @@
+{{- include "stress-test-addons.deploy-job-template.from-pod" (list . "stress.python-eh-stress") -}}
+{{- define "stress.python-eh-stress" -}}
+metadata:
+  labels:
+    testName: "deploy-python-eh-stress"
+    testInstance: "deploy-python-eh-stress-instance"
+    chaos: "true"
+spec:
+  containers:
+    - name: python-eh-stress
+      image: stresstestregistry.azurecr.io/yuling/py-eh-stress:v2
+      {{- include "stress-test-addons.container-env" . | nindent 6 }}
+      command: ['bash', '-c', 'python3 azure_eventhub_producer_stress.py & python3 azure_eventhub_consumer_stress_sync.py']
```

I would swap these two lines, to keep template includes separate from user editable fields.

```suggestion
      command: ['bash', '-c', 'python3 azure_eventhub_producer_stress.py & python3 azure_eventhub_consumer_stress_sync.py']
      {{- include "stress-test-addons.container-env" . | nindent 6 }}
```